src/main/java/com/socialmedia/controller/PostController.java

```diff
@@ -27,4 +30,10 @@ public PostController(PostMapper postMapper) {
 
     return ResponseEntity.ok(postMapper.getAll());
   }
+
+  @PostMapping
+  public ResponseEntity<PostDtoIn> create(@RequestBody PostDtoIn dtoin) {
+    PostDtoIn input = dtoin;
+    return ResponseEntity.ok(dtoin);
+  }
 }
```

Comment:
I understand that I brake the process, but these changes will lead to conflict at our code.

Reply:
I needed to have endpoints to check the functionality, also I changed DtoIn. But you can just override these dummies with your own code

Reply:
OK